lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -529,9 +509,6 @@ namespace Js
         }
         JavascriptLibrary* library = scriptContext->GetLibrary();
 
-        // Ensure JsBuiltIns are initialized before initializing Intl which uses some of them.
-        library->EnsureBuiltInEngineIsReady();
-
```

Comment:
Due to some slightly awkward logic this function to initialise native methods needed for Intl is called when loading any JsBuiltins.

Previously it also called the function to load all JsBuilltins - no longer want that to happen as don't want loading say the Math_object to trigger loading the Array_prototype.